overops-functions/src/main/java/com/takipi/udf/microsoftteams/MicrosoftTeamsChannelRequest.java

```diff
@@ -0,0 +1,186 @@
+package com.takipi.udf.microsoftteams;
+
+import com.google.gson.Gson;
+import com.takipi.api.client.data.event.Location;
+import com.takipi.api.client.result.EmptyResult;
+import com.takipi.api.client.result.event.EventResult;
+import com.takipi.api.core.consts.ApiConstants;
+import com.takipi.api.core.request.intf.ApiPostRequest;
+import com.takipi.udf.microsoftteams.card.*;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MicrosoftTeamsChannelRequest implements ApiPostRequest<EmptyResult> {
+
+    public static final String THEME_COLOR = "ffc000";
+    private String url;
```

Comment:
Members of this class can probably be `final`

Reply:
done